src/Tizen.NUI.Components/Controls/ItemsView/CollectionView.cs

```diff
@@ -0,0 +1,815 @@
+
+            switch (SelectionMode)
+            {
+                case ItemSelectionMode.Single:
+                    if (item.BindingContext == SelectedItem) item.IsSelected = true;
+                    break;
+
+                case ItemSelectionMode.Multiple:
+                    if (SelectedItems!=null && SelectedItems.Contains(item.BindingContext)) item.IsSelected = true;
+                    break;
+                case ItemSelectionMode.None:
+                    item.IsSelectable = false;
+                    break;
+            }
+            
+            
+            return item;
+        }
+
+        // Unrealize and caching the item.
+        internal override void UnrealizeItem(ViewItem item, bool recycle = true)
+        {
+            if (item == Header)
+            {
+                item.Hide();
+                Console.WriteLine("Header is Hide!");
+                return;
+            }
+
+            if (item == Footer)
+            {
+                item.Hide();
+                return;
+            }
+
+            item.IsSelected = false;
+            base.UnrealizeItem(item, recycle);
+        }
+
+        /// <summary>
+        /// update selected items list in multiple selection.
+        /// </summary>
+        /// <since_tizen> 6.5 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+		public void UpdateSelectedItems(IList<object> newSelection)
+		{
+			var oldSelection = new List<object>(SelectedItems);
+
+			_suppressSelectionChangeNotification = true;
+
+			SelectedItems.Clear();
+
+			if (newSelection?.Count > 0)
+			{
+				for (int n = 0; n < newSelection.Count; n++)
+				{
+					SelectedItems.Add(newSelection[n]);
+				}
+			}
+
+			_suppressSelectionChangeNotification = false;
+
+			SelectedItemsPropertyChanged(oldSelection, newSelection);
+		}
+
+        /// <summary>
+        /// Internal selection callback.
+        /// </summary>
+        /// <since_tizen> 6.5 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+		protected virtual void OnSelectionChanged(SelectionChangedEventArgs args)
+		{
+            //Selection Callback
+		}
+
+		static object CoerceSelectedItems(BindableObject bindable, object value)
+		{
+			if (value == null)
+			{
+				return new SelectionList((CollectionView)bindable);
+			}
+
+			if (value is SelectionList)
+			{
+				return value;
+			}
+
+			return new SelectionList((CollectionView)bindable, value as IList<object>);
+		}
+
+		internal void SelectedItemsPropertyChanged(IList<object> oldSelection, IList<object> newSelection)
+		{
+			if (_suppressSelectionChangeNotification)
+			{
+				return;
+			}
+
+            foreach(View content in ContentContainer.Children)
```

Comment:
foreach 뒤에 1공백 추가 부탁드립니다.
```suggestion
            foreach (View content in ContentContainer.Children)
```